File: lista_1/xadrez.py
# opcao 1 de codigo

L = int(input())
C = int(input())
if L % 2 != 0 and C % 2 != 0 or L % 2 == 0 and C % 2 == 0:
  print('1')
elif L % 2 == 0 and C % 2 != 0 or L % 2 != 0 and C % 2 == 0:
  print('0')


# opcao 2 de codigo

L = int(input())
C = int(input())
H = L + C
if H % 2 == 0:
  print('1')
elif H % 2 != 0:
  print('0')

File: lista_1/Xadrez.java
package beecrowd_Java\lista_1;

import java.io.IOException;
import java.util.*;

public class Xadrez {
    
    public static void main(String[] args) throws IOException {

        Scanner input = new Scanner(System.in);
        
        int linhas = input.nextInt();
        int colunas = input.nextInt();
        int cor = 0;
        
        if ((linhas + colunas) % 2 == 0) {
            cor = 1;
        } else if ((linhas + colunas) % 2 != 0) {
            cor = 0;
        }
        
        System.out.println(cor);

    }

}